reader/__init__.py

```diff
@@ -8,18 +8,15 @@
 
 See https://github.com/realpython/reader/ for more information
 """
-import importlib_resources as _resources
-try:
-    from configparser import ConfigParser as _ConfigParser
-except ImportError:  # Python 2
-    from ConfigParser import ConfigParser as _ConfigParser
-
+from configparser import ConfigParser
+from importlib import resources
 
 # Version of realpython-reader package
 __version__ = "1.0.0"
```

Comment:
Should we update the version number?

Reply:
Yes, but I usually don't do that in a PR like this. At least in a bigger project, there might be other commits coming in, etc. Instead, I usually bump the version as the first step in the release process.